docs/source/gbd2019_models/risk_exposures/maternal_bmi/index.rst

```diff
@@ -0,0 +1,113 @@
+   * - >=30
+     - Obese
+     - A BMI ≥30 signifies obesity, which is a disease that is largely preventable through lifestyle changes. The costs attributable to obesity are high, not only in terms of premature death and health care, but also in terms of disability and a diminished quality of life.
+
+Risk Exposures Description in GBD/DHS
+----------------------------------------
+
+Maternal underweight is not modeled as a GBD risk factor. Instead, maternal underweight is estimated in GBD through two covariates. 
+
+- Covariate #1252: Age standardized covariate for underweight women of reproductive age (10-54) with underweight for women under age 20 defined by <-2 BMI z scores and for women over age 20 as <17 BMI
+
+- Covariate #1253: Age specific covariate for underweight women. Note: underweight for women under age 20 defined by <-2 BMI z scores and for women over age 20 as <17 BMI.
+
+Demographic and Health Surveys report nationally representative estimates of several parameters related to population, health, and nutrition in low and middle income countries. As part of their estimates of nutrition of children and adults, they report BMI (mean, as well as percent below/above threshold values) among women 15-49 years of age by 5-10 year age group, residence (urban/rural), region, education, and wealth quintile.
+
+We chose to inform the maternal BMI distribution from DHS surveys rather than GBD covariate because:
+
+- The BMI threshold of 18.5 reported in DHS is more commonly used than the BMI threshold of 17 in the GBD covariate
+
+- The DHS surveys are publically available and the GBD covariate estimates and methods are not published
+
+Vivarium Modeling Strategy
+--------------------------
+
+The following risk exposure vivarium modeling strategy is intended to apply to mothers or children separately rather than to mother-child dyads.
+
+Maternal BMI should be implemented as a **dichotomous** risk exposure. The exposed group is BMI<18.5 and the TMREL is BMI>=18.5. 
+
+When modeling children, the exposure should be assigned at birth or upon initialization at the start of the simulation according to the probability listed in the table below. The TMREL exposure category should be assigned according to the probability equal to 1 minus the exposed probability.
+
+.. list-table:: Risk exposure values
+   :header-rows: 1
+
+   * - Location
+     - Population
+     - Exposure category
+     - Value
+     - Note
+     - Source
+   * - Ethiopia
+     - General
+     - Exposed (BMI<18.5)
+     - 0.224 (95% CI: 0.217, 0.231)
+     - Assume normal distribution of uncertainty clipped at 0 and 1. Confidence interval calculated as :math:`\pm 1.96 \times \sqrt{\frac{p \times (1 - p)}{n}}`
+     - Ethiopia 2016 DHS
```

Comment:
I suggest to attach the link/reference for location-/year-specific DHS surveys. (e.g., internal link of GHDx)

Reply:
Great suggestion, thank you! just added :)